onnxruntime/python/tools/transformers/onnx_exporter.py

```diff
@@ -347,7 +350,7 @@ def validate_and_optimize_onnx(model_name,
 
 def export_onnx_model_from_pt(model_name, opset_version, use_external_data_format, model_type, model_class, cache_dir,
                               onnx_dir, input_names, use_gpu, precision, optimize_onnx, validate_onnx,
-                              use_raw_attention_mask, overwrite, model_fusion_statistics):
+                              use_raw_attention_mask, overwrite, model_fusion_statistics, no_opt=False):
```

Comment:
I saw names of bypass_optimization and no_opt. It might be better to use same parameter name everywhere to be consistent.

Also, there is parameter optimize_onnx (True will use python script for optimization, False will use ORT to optimize). Shall we consolidate them. For example, change optimize_onnx to have Enum value: (ByOptimizer, ByOrt, NoOpt)